api/trading_data_v2.go

```diff
@@ -68,6 +68,65 @@ type tradingDataServiceV2 struct {
 	transfersService          *service.Transfer
 }
 
+func (t *tradingDataServiceV2) ListAccounts(ctx context.Context, req *v2.ListAccountsRequest) (*v2.ListAccountsResponse, error) {
+	if t.accountService == nil {
```

Comment:
We should add metrics to all V2 apis as per V1

Reply:
good point, done